google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/Field.java

```diff
@@ -244,6 +293,21 @@ public PolicyTags getPolicyTags() {
     return policyTags;
   }
 
+  /** Returns the max length for the field. */
```

Comment:
Maybe augment the javadocs for the public getters with the information about what types each of these relates to, but no need to be as exhaustive as the builder's setters.

Reply:
Makes sense. Update Javadoc.